modules/data.atmosphere/R/download.NEONmet.R

```diff
@@ -0,0 +1,486 @@
+##' Download NEON Site Met CSV files
+##'
+##' @name download.NEONmet
+##' @title download.NEONmet
```

Comment:
I think this practice is a legacy of an earlier version of ROxygen yelling at us when these tags were not explicit. If this is done automagically now I'm more than happy to see this updated.